style.css

```diff
@@ -0,0 +1,248 @@
+* {
+    margin: o;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Raleway', sans-serif;
+    appearance: none;
+    -webkit-appearance:none;
+}
+
+body {
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+fieldset {
+    max-width: 400px;
+    padding: 20px;
+    border: 2px solid pink;
+}
+
+fieldset:hover {
+    border: 2px solid rgb(248, 63, 94);
+}
+
+legend {
+    font-size: 24px;
+    text-align: center;
+    color: rgb(248, 63, 94);
+}
+
+p {
+    margin-bottom: 5px;
+    color: rgb(248, 63, 94);
+    text-align: center;
+}
+
+
+/*** TEXTFIELD ***/
+
+
+.textfield-container {
+    display: block;
+    position: relative;
+    margin: 5px;
+    font-size: 16px;
+}
+
+.custom-textfield {
+    position: absolute;
+    top: 0%;
+    left: 0%;
+    cursor: pointer;
+}
+
+input[type=name]{
+    width: 100%;
+    height: 30px;
+    margin: 5px 0 0 -2px;
+    border: none;
+    background-color:  rgb(255, 230, 234);
+}
+
+input[type=email] {
+    width: 100%;
+    height: 30px;
+    margin: 5px 0 0 -2px;
+    border: none;
+    background-color:  rgb(255, 230, 234);
+}
+
+
+/*** RADIO BUTTON ***/
```

Comment:
Great with comments! And GREAT job with both the radio buttons and the checkbox 🥳